docs/04-logging.md

```diff
@@ -28,7 +28,7 @@ class Exp1(SimpleExperiment):
     return e
```

Comment:
yeah it should be removed: you only need to specify a return value when short circuiting or disabling exposure logging.